www/klei00/semester_project_2/js/main.js

```diff
@@ -143,25 +144,191 @@ var addToLibrary = function (book, author, url) {
         }
     }
     var categoryCell = $('<div>').addClass('table-cell').text(category);
+
+    var star1 = $('<span>').addClass('star').text('*');
+    var star2 = $('<span>').addClass('star').text('*');
+    var star3 = $('<span>').addClass('star').text('*');
+    var star4 = $('<span>').addClass('star').text('*');
+    var star5 = $('<span>').addClass('star').text('*');
+
+    var ratingCell = $('<div>').addClass('table-cell').addClass('rating').append(star1).append(star2).append(star3).append(star4).append(star5);
     var deleteCell = $('<div>').addClass('table-cell').addClass('delete').text('Odebrat');
     var idCell = $('<div>').addClass('table-cell').addClass('id').text(book.id);
-    var newRow = $('<div>').addClass('table-row').append(imageCell).append(nameCell).append(authorCell).append(categoryCell).append(yearCell).append(deleteCell).append(idCell);
+    var newRow = $('<div>').addClass('table-row').append(imageCell).append(nameCell).append(authorCell).append(categoryCell).append(yearCell).append(ratingCell).append(deleteCell).append(idCell);
     var tableHead = $('#table-head');
     if (tableHead.hasClass('closed')) {
         tableHead.removeClass('closed');
     }
     $('#my-library').append(newRow);
 
     deleteCell.click(function () {
+        localStorage.removeItem(book.id);
         newRow.remove();
         if ($('.id').length === 1) {
             tableHead.addClass('closed');
         }
         form.submit();
     });
+
+    star1.click(function () {
+        if (!star1.hasClass('checked')) {
```

Comment:
to hodnoceni se neresi takto, porusujes [pravidlo DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself).
reseni by melo byt nejak takto:
```
pri kliknuti na jednu - jakoukoliv hvezdicku proved nasledujici
1. zjistit hodnotu hvezdicky (pri vytvareni hvezdicky prirad ji nejaky data atribut)
2. vybarvit vsechny hvezdicky nalevo vcetne tuto (.nextAll())
3. odbarvit vsechny hvezdicky napravo (.prevAll())
4. update hodnoceni ulozene knizky v localStorage

pri renderovani hodnoceni
1. zjistit hodnoceni
2. vybarvit n-tou hvezdicku a vsechny nalevo
```

Reply:
Opraveno.